OGP1617-Asteroids/src/asteroids/model/programs/expressions/EntityLiteral.java

```diff
@@ -0,0 +1,34 @@
+package asteroids.model.programs.expressions;
+import asteroids.model.Entity;
+
+
+public abstract class EntityLiteral extends Type{
+	
+		public EntityLiteral(Entity entity) {
+			this.entity = entity;
+		}
+
+		public Entity getEntity() {
+			return entity;
+		}
+		
+		private final Entity entity;
+		
+		@Override
+		public boolean equals(Object other) {
+			if(other == null)
+				return false;
+			if(this.getClass() != other.getClass())
+				return false;
+			return this.equals(other);
+		}
```

Comment:
in deze equals zou ik gewoon this.equals(other) zetten dat is dan de default equals voor gewone objecten